### ISSUE TITLE
MacOs Environment variables remplacement failed

### ISSUE THREAD

**Issue by jburdy** [influxdata/telegraf#8484]
The environment variables remplacement does not work in MacOS !?

### Relevant telegraf.conf:
```toml
[global_tags]
  ## Environment variables can be used as tags, and throughout the config file
  user = "${USER}"
```

### System info:
MacOs 11.0.1
Telegraf 1.16.2 (from HomeBrew)

### Steps to reproduce:

1- start telegraf (as root)
2- Check influxDB, the tags are user=${USER} or user=$USER

### Expected behavior:

user="nameOfTheUser"

### Actual behavior:

user=${USER} or user=$USER

### Additional info:

I'm using 
sudo brew services start telegraf
to launch telegraf. 

I have test with:
user = "${USER}"
user = "$USER"

**Comment by Jagularr** [comment]
Hi, after testing this I found that the issue doesn't occur when running Telegraf directly, but only when I ran through "sudo brew services start telegraf" was the issue replicated. I believe it has something to do with homebrew running the service as a different user/environment variables, and for some reason that $USER variable doesn't exist in that context. While I wasn't able to find a quick way to make it work this way, I'm hoping there's a way out there to preserve your user environment variable specifically with brew services.
